cmd/ocm/create/cluster/cmd.go

```diff
@@ -52,6 +52,12 @@ const (
 	gcpTermsAgreementNonInteractiveError = "Review and accept Google Terms and Agreements on " +
 		gcpTermsAgreementsHyperlink + ". Set the flag --marketplace-gcp-terms to true " +
 		"once agreed in order to proceed further."
+
+	privateFlag            = "private"
+	vpcNameFlag            = "vpc-name"
+	controlPlaneSubnetFlag = "compute-subnet"
+	computePlaneSubnetFlag = "control-plane-subnet"
```

Comment:
These two const names are mixed up for their values and not used in defining the flag.

Reply:
If we are adding consts for these, we should replace the hard-coded values.

Reply:
I just replaced in the code I  was working on but yes I replace all